package.json

```diff
@@ -5,10 +5,11 @@
   "description": "setup ruby action",
   "main": "lib/setup-ruby.js",
```

Comment:
bump version 1.0.0 ?

Reply:
the package version here has no bearing but OK